client/client.go

```diff
@@ -16,67 +16,31 @@ package client
 
 import (
 	"context"
-	"fmt"
-	"strings"
 
 	"github.com/tigrisdata/tigris-client-go/config"
 	"github.com/tigrisdata/tigris-client-go/driver"
 )
 
 // Client is the interface for a Tigris client.
 type Client interface {
-	// Database returns a Database interface for the provided
-	// database name.
-	//
-	// TODO: Return bool/error if the database exists or not
-	//       or should the client just get the error when they
-	//       try to use it? Latter is more performant.
-	Database(name string) Database
-
-	// CreateDatabaseIfNotExist creates a database if it doesn't
-	// already exist.
-	CreateDatabaseIfNotExist(
-		ctx context.Context,
-		db string,
-		options ...*driver.DatabaseOptions,
-	) error
+	UseDatabase(name string) *Database
 }
 
 type client struct {
 	driver driver.Driver
 }
 
-func NewClient(
-	ctx context.Context,
-	config *config.Config,
-) (Client, error) {
+// NewClient connects to the Tigris using provided config
+func NewClient(ctx context.Context, config *config.Config) (Client, error) {
 	d, err := driver.NewDriver(ctx, config)
 	if err != nil {
 		return nil, err
 	}
-	return &client{
-		driver: d,
-	}, nil
+	return &client{driver: d}, nil
 }
 
-func (c *client) Database(name string) Database {
+// UseDatabase returns new Database object, which is used to get access
+// to collections and document APIs of the Database with the name 'name'
+func (c *client) UseDatabase(name string) *Database {
```

Comment:
is UseDatabase creating the database as well?

Reply:
No. It doesn't.